docs/introduction/understanding-docker.md

```diff
@@ -21,7 +21,7 @@ running code.
 Docker does this by combining a lightweight container virtualization platform
 with workflows and tooling that help you manage and deploy your applications.
 
-At its core, Docker provides a way to run almost any application securely
+At its core, Docker provides a way to run almost any applications securely
```

Comment:
Agree on singular application.

"securely isolated" is what reads weird to me. Maybe "isolated and secure"? Maybe drop `secure` totally. I realize we're trying to play up the security that isolation provides, and docker provides the isolation.